.changeset/wild-ads-pull.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/plugin-api-docs': patch
+'@backstage/plugin-catalog': patch
```

Comment:
plugin-catalog is at version 1.x so new features should be `minor`. I suggest you split this up into two changeset, they can contain the same text but one being patch and the other minor.

Reply:
Sounds good, thanks Johan!